src/common/cli_wrapper.h

```diff
@@ -54,20 +77,25 @@ class CLIWrapper {
 
   static std::string failureMessage(const CLI::App *app, const CLI::Error &e);
 
+  // Extract an option name from comma-separated list of command-line arguments,
+  // e.g. 'help' from '--help,-h'
   std::string keyName(const std::string &args) const {
-    return std::get<1>(CLI::detail::get_names(CLI::detail::split_names(args)))
-        .front();
+    // re-use existing functions from CLI11 to keep option names consistent
+    return std::get<1>(CLI::detail::get_names(CLI::detail::split_names(
+                           args)))  // get long names only
+        .front();                   // get first long name
```

Comment:
(Can there be multiple long names? Should there be? If not, maybe add to comment "(there should really only be one)")

Reply:
That's the API of the library: https://github.com/CLIUtils/CLI11
Couldn't find if it support multiple long options (why not?), but no reason to dispute this.